### PR TITLE
[TW-757] Remove creating a monitor from an API

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Removed text about creating a monitor from an API (no longer possible in v10)
- Updated screenshots
- Minor page refresh

### CODE REVIEW

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md [hunk 1]
```diff
@@ -108,7 +107,7 @@ Static IPs are available on Postman Professional and Enterprise plans when you m
 
 ### Getting monitor notifications
 
-When configuring a monitor, you can choose to receive email notifications for monitor run failures and errors. You can also add other email recipients to receive notifications
+When configuring a monitor, you can choose to receive email notifications for monitor run failures and errors. You can add up to five email recipients to receive notifications
```
Comment (jkonrath-postman) [inline src/pages/docs/monitoring-your-api/setting-up-monitor.md:110]:
```suggestion
When configuring a monitor, you can choose to receive email notifications for monitor run failures and errors. You can add up to five email recipients to receive notifications.
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/monitoring-your-api/setting-up-monitor.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/monitoring-your-api/setting-up-monitor.md:110]: Thanks for catching that!

**Pull request merged**